Tests/Functional/CompositeRepositoryTest.php

```diff
@@ -11,8 +11,10 @@
 
 namespace Symfony\Cmf\Bundle\ResourceBundle\Tests\Functional;
 
-class CompositeRepositoryTest extends PhpcrRepositoryTestCase
+class CompositeRepositoryTest extends RepositoryTestCase
```

Comment:
I fixed the abstraction now. Both (or all three) repository tests do have one abstract method with the `setUp()`, cause even `CompositeRepositoryTest` needs some nodes/documents prepared:

```

        stuff:
            type: composite
            mounts:
                - { repository: test_repository_phpcr_odm, mountpoint: /content }
                - { repository: articles, mountpoint: /articles }
```
It works in that case `doctrine_phpcr_odm` type. So the composite repository shared the `setUp()` before my change and does it now. The name the extended name is cleare now and both phpcr repositories have its own abstraction to collect common methods now.